pkg/util/netlink.go

```diff
@@ -101,14 +101,27 @@ func LinkDelete(link string) error {
 	return err
 }
 
-// Listen for events on a specific interface and callback if any. The interface
-// does not have to exist. Use the stop channel to stop listening.
+// OnLinkEvent upkeeps a subscription to netlink events and callbacks for any
+// event on a specific interface.
+// The subscription might temporarily fail. On resubscription, the callback is
+// invoked to cover for events that might have been missed on that time. That
+// means some spurious callbacks unrelated to the interface might happen and
+// the callee should account for it. For convenience, to avoid losing any
```

Comment:
'caller' ?

Reply:
Yes, the caller of onLinkEvent. Previously I used callee but that was a mistake. Whats's the concern?

Reply:
The concern is that I'm seeing 'callee' in commit https://github.com/kubevirt/macvtap-cni/pull/11/commits/f3874889c64e6ec84b0fcd03b48ef1047af75e7a .

Reply:
Fixed from that change.